### PR TITLE
Last minute bugfixes

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Critical bugfix

## Pull request description

Fixes some bootstrap stuff in the frontend
 swiftmail not being able to switch to smtp
the mailmotor commands where listening for the wrong classes
fixes te test mail settings call

### CODE REVIEW

File: src/Common/Core/Form.php [hunk 1]
```diff
@@ -20,18 +31,178 @@ class Form extends \SpoonForm
...
+     * @var BackendUrl|FrontendUrl
      */
     protected $url;
 
     /**
+     * @param string $name Name of the form
+     * @param string|null $action The action (URL) whereto the form will be submitted, if not provided it
+     *                            will be auto generated.
+     * @param string|null $method The method to use when submitting the form, default is POST.
+     * @param string $hash     The id of the anchor to append to the action-URL.
```
Comment (StijnVrolijk) [inline src/Common/Core/Form.php]: You have some extra spaces here :)

File: src/Frontend/Core/Layout/Templates/FormLayout.html.twig [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {% extends 'bootstrap_3_layout.html.twig' %}
 
 {%- block form_label -%}
+  <h1>this is a label</h1>
```
Comment (StijnVrolijk) [inline src/Frontend/Core/Layout/Templates/FormLayout.html.twig]: 🤔

File: src/Common/Core/Form.php (hunk 1 above)
Comment (carakas) [inline src/Common/Core/Form.php]: fixed in 4fa8c664c

Comment (carakas) [inline src/Common/Core/Form.php]: fixed in 4fa8c664c

File: src/Frontend/Core/Layout/Templates/FormLayout.html.twig (hunk 2 above)
Comment (carakas) [inline src/Frontend/Core/Layout/Templates/FormLayout.html.twig]: fixed in 05a753836

**Pull request merged**